README.md

```diff
@@ -26,34 +26,47 @@
 
 **nf-core/references** is a bioinformatics pipeline that build references, for multiple use cases.
 
-It is primarily designed to build references for common organisms and store it on [AWS iGenomes](https://github.com/ewels/AWS-iGenomes/).
+For most common organisms the plan is to build the references and store them in the cloud, similar to [AWS iGenomes](https://github.com/ewels/AWS-iGenomes/).
 
-From a fasta file, it will be able to build the following references:
+![nf-core/references metro map](docs/images/nf-core-references_metro_map_color.png)
+
+## DNASEQ
+
+For DNASEQ pipelines (ie [nf-core/sarek](https://nf-co.re/sarek)), needing only a fasta file, it will be able to build the following references:
 
-- Bowtie1 index
-- Bowtie2 index
```

Comment:
Why no bowtie*? And why no `bwa-aln`? Those modules exist, and people still use those a lot *ahem* _eager/mag/taxprofiler etc_

Reply:
out of current scope, but that can be shipped quite quickly once we have a first release